README.md

```diff
@@ -240,7 +240,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-   http://www.apache.org/licenses/LICENSE-2.0
+   https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This link comes from the boilerplate text that our license requires us to include (See [LICENSE](https://github.com/Kong/kong/blob/master/LICENSE#L190-L202)). I am not sure of the consequences if we update it here but the license specifies another link, nor even if we can update the link instructed by the license itself.